src/Sylius/Bundle/CoreBundle/Behat/CoreContext.php

```diff
@@ -72,7 +72,9 @@ public function storeHasDefaultConfiguration()
         $channel->setName('Default');
         $channel->setUrl('http://example.com');
         $channel->addCurrency($currency);
```

Comment:
Btw. shouldn't currency and locale be added automatically when setting them as default?